account_tax_one_vat/__manifest__.py

```diff
@@ -0,0 +1,19 @@
+# Copyright 2020 Camptocamp SA
+# Copyright 2023 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html)
+
+{
+    "name": "Account Tax One VAT",
+    "summary": "Allow only the selection of one VAT Tax.",
+    "version": "16.0.1.0.0",
+    "author": "Camptocamp, Acsone, Odoo Community Association (OCA)",
+    "license": "AGPL-3",
+    "category": "account",
+    "depends": ["account", "sale", "purchase"],
```

Comment:
I think that you could make a generic module `account_tax_one_vat` and some glue modules `account_tax_one_vat_purchase` `account_tax_one_vat_sale`.
That should not ask a lot of work, and allow better modularity.